runtime/sema/authaccount.cdc

```diff
@@ -83,7 +83,7 @@ access(all) struct AuthAccount {
     /// The given type must not necessarily be exactly the same as the type of the copied structure.
     ///
     /// The path must be a storage path, i.e., only the domain `storage` is allowed.
-    access(all) fun copy<T: AnyStruct>(from: StoragePath): T?
+    access(all) view fun copy<T: AnyStruct>(from: StoragePath): T?
```

Comment:
Can some of these other functions be `view` as well?
e.g: `borrow`, `check`,  `getCapability`, `getLinkTarget` - non of them change the state I guess

Reply:
lgtm!